qiskit-superstaq/qiskit_superstaq/__init__.py

```diff
@@ -13,6 +13,7 @@
 from .superstaq_backend import SuperstaQBackend
 from .superstaq_job import SuperstaQJob
 from .superstaq_provider import SuperstaQProvider
+from .validation import validate_integer_param, validate_qiskit_circuits, validate_target
```

Comment:
nit: it looks like we only ever access these by their full path (e.g. `qss.validation.validate_integer_param`), so no reason to import them directly at this level
```suggestion
```
(or alternatively we should just use e.g. `qss.validate_integer_param` everywhere else)